drivers/wifi/nrf700x/Kconfig

```diff
@@ -74,10 +74,23 @@ config NRF_WIFI_IF_AUTO_START
 
 config NRF_WIFI_PATCHES_BUILTIN
 	bool "Store nRF700x FW patches as part of the driver"
+	depends on !NRF_WIFI_PATCHES_EXT_FLASH_STORE
 	help
 	  Select this option to store nRF700x FW patches as part of the driver.
 	  This option impacts the flash memory size of the application.
-	default y
+	default y if !NRF_WIFI_PATCHES_EXT_FLASH_STORE
+
+choice NRF_WIFI_PATCHES_EXT_FLASH_SUPPORT
+	bool "Store nRF700x FW patches in external flash"
```

Comment:
I'd use "external memory" and not use the term Flash

Reply:
For the KCONFIG options it does not matter much, though. But the text could use "external memory"..